native/Makefile

```diff
@@ -111,6 +111,18 @@ endif
 STRIP=strip -x
 # end defaults
 
+
+OS_PLATFORM = $(shell uname -m)
+ifeq ($(OS_PLATFORM), mips64)
+    LDFLAGS += -Wl,-z,noexecstack
+endif
+
+OS_PLATFORM = $(shell uname -m)
+ifeq ($(OS_PLATFORM), loongarch64)
+    LDFLAGS += -Wl,-z,noexecstack
+endif
```

Comment:
Any reason for these additions? Given, that this PR is about loongarch, I'm surprised to see changes for mips64.

Reply:
> This look broken as for example android and at least windows aarch64 are crossbuild, so the local platform should not affect the build. Also the other architectures have no similar treatment and mips64 should not even be touched, as this is targetted for loongarch.

My fault. I will revise it